ui/app/rif/pages/addNewMulticryptoAddress/domain-add-new-network-page.css

```diff
@@ -0,0 +1,77 @@
+.custom-select{
```

Comment:
General comment, it's better if you use relative units on this file instead of fixed pixels, i know this is a plugin but it's always better if we use relative units instead of fixed units.